squash_wip.go

```diff
@@ -114,15 +114,16 @@ func replaceFileContents(fileName string, replacer Replacer) {
 
 func commentWipCommits(input string, configuration Configuration) string {
 	var result []string
-	isWipCommitBlock := false
-	for _, line := range strings.Split(input, "\n") {
+	ignoreBlock := false
+	lines := strings.Split(input, "\n")
+	for idx, line := range lines {
 		if configuration.isWipCommitMessage(line) {
-			isWipCommitBlock = true
-		} else if line == "" {
-			isWipCommitBlock = false
+			ignoreBlock = true
+		} else if line == "" && len(lines) > idx+1 && strings.HasPrefix(lines[idx+1], "#") {
```

Comment:
This condition is hard to make sense of. I think we should move it in it's own function with a descriptive name.

Reply:
please have a look at the newly extracted function isNextLineComment. Is that fine for you?